bin/update-yarn.sh

```diff
@@ -21,7 +21,9 @@ PLUGINS=$(awk '{ if ($1 == "spec:") print $2 }' .yarnrc.yml)
 echo "$PLUGINS" | xargs -n1 -t corepack yarn plugin remove
 
 cp package.json .yarn/cache/tmp.package.json
```

Comment:
I'm not sure how nice it is to use yarn's cache dir, but probably they will never put a file tmp.package.json file there, but you never know 😄